lighthouse-core/test/audits/legacy-javascript-test.js

```diff
@@ -144,11 +144,12 @@ describe('LegacyJavaScript audit', () => {
       'String.prototype[\'repeat\'] = function() {}',
       'Object.defineProperty(String.prototype, "repeat", function() {})',
       'Object.defineProperty(String.prototype, \'repeat\', function() {})',
-      'Object.defineProperty(window, \'WeakMap\', function() {})',
+      // Currently are no polyfills that declare a class. May be in the future.
+      // 'Object.defineProperty(window, \'WeakMap\', function() {})',
       '$export($export.S,"Object",{values:function values(t){return i(t)}})',
-      'WeakMap = function() {}',
-      'window.WeakMap = function() {}',
-      'function WeakMap() {}',
+      // 'WeakMap = function() {}',
```

Comment:
should we delete these or are these comments important?

Reply:
Did you see my comment a few lines up

Reply:
I did, I just didn't make the connection :)

maybe just group all 4 of them together with that comment?